lib/find-python.js

```diff
@@ -426,24 +435,27 @@ class PythonFinder {
         )
         this.log.silly('execFile: opts = ', JSON.stringify(opts, null, 2))
 
-        //* assume that user use utf8 compatible termnal
-
-        //* prosible outcomes with error messages (err.message, error.stack, stderr)
-        //! on windows:
+        //* prosible outcomes with error messages on windows (err.message, error.stack?, stderr)
         // issue of encoding (garbage in terminal ) when 866 or any other locale code
         // page is setted
         // possible solutions:
-        // use "cmd" command with flag "/U" and "/C" (for more informatiom help cmd)
+        // 1. let it be as it is and just warn user that it should use utf8
+        // (already done in this.catchErros's info statement)
+        // 2. somehow determine user's terminal encdoing and use utils.TextDecoder
+        // with getting raw buffer from execFile.
+        // Requires to correct error.message becouse garbage persists there
+        // 3. Forse user's terminal to use utf8 encoding via e.g. run "chcp 65001". May broke some user's programs

```

Comment:
```suggestion
        // 1. leave it as is and just warn the user that it should use utf8
        // (already done in this.catchError's info statement)
        // 2. somehow determine then user's terminal encoding and use utils.TextDecoder
        // with the raw buffer from execFile.
        // Requires to correct error.message because garbage persists there
        // 3. Force the user's terminal to use utf8 encoding via e.g. run "chcp 65001". May break some user's programs
```